### DISCUSSION TITLE
Flutter macos publishing not working

### DISCUSSION THREAD

**Discussion by IsmailAlamKhan** [codemagic-ci-cd/codemagic-docs#1902] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

1. Make a flutter macos app workflow on codemagic which will build and publish as well.
2. Run the build.

### Expected results

The build should pass and it should be published to MacOS store

### Actual results

I get an error on the publishing step.
```
 Error: ERROR: [ContentDelivery.Uploader] Asset validation failed (90296) App sandbox not enabled. The following executables must include the "com.apple.security.app-sandbox" entitlement with a Boolean value of true in the entitlements property list: [( "com.neurocareai.neuroich.admin.pkg/Payload/NeuroICH Admin.app/Contents/MacOS/NeuroICH Admin" )] Refer to App Sandbox page at https://developer.apple.com/documentation/security/app_sandbox for more information on sandboxing your app. (ID: a56dd139-e4fe-4fb6-87bf-aad067da412b)
 ```
 
 Even though I have enabled App Sandbox.

### Build id (optional)

64b5554146dccc76c43e7e05

**Comment by himesh-cm** [comment]
Hi @IsmailAlamKhan, Can you please enable the App sandbox as shown [here](https://developer.apple.com/documentation/xcode/configuring-the-macos-app-sandbox/) in the doc and try re-running the build?

**Reply by IsmailAlamKhan** [reply]
Yes that's what I have been doing, manually running build and publish from local machine works

**Reply by himesh-cm** [reply]
Sorry, can you send me a screenshot of your Xcode? It will help me understand better.

**Comment by himesh-cm** [comment]
I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.
